static/templates/compose_control_buttons.hbs

```diff
@@ -13,7 +13,8 @@
     <div class="{{#if message_id}}hide-lg{{else}}hide-sm{{/if}}">
         {{> compose_control_buttons_in_popover}}
     </div>
-    <a role="button" class="compose_control_button compose_draft_button hide-sm" tabindex=0 href="#drafts" data-tippy-content="{{t 'Drafts' }}">
+    <a role="button" class="compose_control_button compose_draft_button hide-sm" tabindex=0 href="#drafts" data-tooltip-template-id="compose_draft_tooltip">
+    <template id="compose_draft_tooltip">{{t 'Drafts' }} <span class="hotkey-hint">(d)</span></template>
         {{t 'Drafts' }}
     </a>
```

Comment:
Please put the `<template>` next to the `<a>` tag and not inside it, as that's how it's done in other parts of the codebase:

```html
    <a role="button" class="compose_control_button compose_draft_button hide-sm" tabindex=0 href="#drafts" data-tooltip-template-id="compose_draft_tooltip">
        {{t 'Drafts' }}
    </a>
    <template id="compose_draft_tooltip">{{t 'Drafts' }} <span class="hotkey-hint">(d)</span></template>
```